src/main/java/com/stripe/param/TokenCreateParams.java

```diff
@@ -0,0 +1,2512 @@
+      this.last4 = last4;
+      this.metadata = metadata;
+      this.name = name;
+      this.number = number;
+      this.numberLooksValid = numberLooksValid;
+      this.numberLuhnValid = numberLuhnValid;
+      this.object = object;
+      this.threeDSecure = threeDSecure;
+      this.tokenizationMethod = tokenizationMethod;
+    }
+
+    public static Builder builder() {
+      return new com.stripe.param.TokenCreateParams.CreditCard.Builder();
+    }
+
+    public static class Builder {
+      private String addressCity;
+
+      private String addressCountry;
+
+      private String addressLine1;
+
+      private String addressLine2;
+
+      private String addressState;
+
+      private String addressZip;
+
+      private String cardType;
+
+      private String currency;
+
+      private String customer;
+
+      private Boolean defaultForCurrency;
+
+      private String expMonth;
+
+      private String expYear;
+
+      private String last4;
+
+      private Map<String, String> metadata;
+
+      private String name;
+
+      private String number;
+
+      private Boolean numberLooksValid;
+
+      private Boolean numberLuhnValid;
+
+      private ObjectType object;
+
+      private ThreeDSecure threeDSecure;
+
+      private String tokenizationMethod;
+
+      /** Finalize and obtain parameter instance from this builder. */
+      public CreditCard build() {
+        return new CreditCard(
+            this.addressCity,
+            this.addressCountry,
+            this.addressLine1,
+            this.addressLine2,
+            this.addressState,
+            this.addressZip,
+            this.cardType,
+            this.currency,
+            this.customer,
+            this.defaultForCurrency,
+            this.expMonth,
+            this.expYear,
+            this.last4,
+            this.metadata,
+            this.name,
+            this.number,
+            this.numberLooksValid,
+            this.numberLuhnValid,
+            this.object,
+            this.threeDSecure,
+            this.tokenizationMethod);
+      }
+
+      /**
+       * Add all map key/value pairs to `metadata` map. A map is initialized for the first
+       * `put/putAll` call, and subsequent calls add additional key/value pairs to the original map.
+       * See {@link CreditCard#metadata} for the field documentation.
+       */
+      public Builder putAllMetadata(Map<String, String> map) {
+        if (this.metadata == null) {
+          this.metadata = new HashMap<>();
+        }
+        this.metadata.putAll(map);
+        return this;
+      }
+
+      /**
+       * Add a key/value pair to `metadata` map. A map is initialized for the first `put/putAll`
+       * call, and subsequent calls add additional key/value pairs to the original map. See {@link
+       * CreditCard#metadata} for the field documentation.
+       */
+      public Builder putMetadata(String key, String value) {
+        if (this.metadata == null) {
+          t
```

Comment:
This should not be here, there's no 3DS class on token creation and if there is it's some weird internal thing I would say.

Reply:
Removed 3d secured